CHANGES.md

```diff
@@ -79,6 +79,8 @@
 ## Bugfixes
 
 * Fixed X (Java/Python) ([#X](https://github.com/apache/beam/issues/X)).
+* Fixed EventTimeTimer ordering in Prism. ([#32222](https://github.com/apache/beam/issues/32222)).
```

Comment:
this doesn't seem to be in this PR

Reply:
Hmm must be from a HEAD sync. will remove, thanks